react/features/invite/functions.js

```diff
@@ -297,9 +297,10 @@ export function invitePeopleAndChatRooms( // eslint-disable-line max-params
  * @returns {boolean} Indication of whether adding people is currently enabled.
  */
 export function isAddPeopleEnabled(state: Object): boolean {
+    const { peopleSearchUrl } = state['features/base/config'];
     const { isGuest } = state['features/base/jwt'];
 
-    return !isGuest;
+    return !isGuest && Boolean(peopleSearchUrl);
```

Comment:
Hum, we use add people dialog for adding people using peopleSearchUrl and/or for jigasi calling. So I don't thing this is correct. Cause you need to enable peopleSearchUrl whenever you have jigasi.

Reply:
See: https://github.com/jitsi/jitsi-meet/blob/master/react/features/toolbox/components/web/Toolbox.js#L1323

We always check for both add people (search in the directory) and dialout (jigasi) AFAICT. If that is not configured it would still work because this would return true: https://github.com/jitsi/jitsi-meet/blob/master/react/features/invite/functions.js#L311